openedx/core/djangoapps/waffle_utils/views.py

```diff
@@ -28,6 +29,8 @@ def get(self, request):
         response['waffle_flags'] = self._get_all_waffle_flags()
         response['waffle_switches'] = self._get_all_waffle_switches()
         response['django_settings'] = self._get_settings_state()
+        response['setting_toggles'] = self._get_setting_toggle_state()
+        response['setting_dict_toggles'] = self._get_setting_dict_toggle_state()
```

Comment:
1. I'd like to combine all three of these under `'django_settings'`, to be more consistent with how we handled waffle.  We used `waffle_flags` for anything that primarily uses a waffle flag as its base, even if it isn't using WaffleFlag, but is directly coming out of the model.  We differentiate using `class: "WaffleFlag"` in the result.
2. Using the pattern in `_get_all_waffle_flags()` for a new `_get_all_django_settings()` function would make the view's implementation more consistent.  It would also simplify some code, as I'll note in further comments.

**Note:** Regarding simpler PR reviews, I think it would be easier going forward to simply have separate PRs for any commit that you think is separate.  We should be able to land smaller PRs faster.  It may make sense to move this toggle state endpoint work to a separate PR, given that I still have feedback on it.

Reply:
@regisb: Reminder: Can you move these toggle state endpoint changes to a separate PR so we can approve/land the settings usage?

Reply:
Right! Sorry about that, I admit that I completely forgot about this PR, and somehow I skipped your comment notifications :sweat_smile: I'll remove the last commit from this PR.